### ISSUE TITLE
0.7.0 release

### ISSUE THREAD

**Issue by GoogleCodeExporter** [VestniK/qremotesignal#20]
```
Version 0.7.0 main target is to produce good documentation for library and qrsc 
tool. Also there 
will be few enhancements in the code itself.

TODO list:
  * Man page for qrsc tool
  * API docs for Doxygen
  * Windows support (already done in svn)
  * Simplify work with QIODevice
```

Original issue reported on code.google.com by `sir.vestnik` on 15 Oct 2009 at 6:52
- Blocked on: #6, #14, #21, #22
